Chat display update!

holistudio opened the pull request:
what is wrong?

holistudio committed:
Navbar with brand icon and toggler (smaller screen)

holistudio committed:
Initial conversation display

holistudio committed:
New placeholder icon

holistudio committed:
Temporarily disabled React app

holistudio committed:
update chat interface

holistudio committed:
Merge branch 'QuestionBranch' into ChatDisplay

holistudio committed:
merge conversation app with interface

holistudio committed:
container class instead of id

holistudio committed:
Merge branch 'isomorph-refactor' into ChatDisplay

holistudio committed:
bootstrap CSS to HTML.js

holistudio committed:
package json git repo URL update

holistudio committed:
webpack config removed htmlweback injection into body

holistudio committed:
update webpack config with css and style loader with module options

holistudio committed:
added style loader dependency

holistudio committed:
updated CSS loader

holistudio committed:
simplified style and css loader setup, removed htmlwebpack for testing

holistudio committed:
experimenting with CSS

holistudio committed:
removed styling css from unnecessary places

holistudio committed:
Update package.json, prestart automatically build:dev

holistudio committed:
package.js prestart fix

holistudio committed:
HTML.js update, favicon added

static files will go directly into build/static/. will look into how webpack can load these automatically later

holistudio committed:
moved logo-placeholder and menu icon to build/static/Navigation

holistudio committed:
Initialize Conversation React Components

holistudio committed:
Brand Component logo png added

holistudio committed:
MenuButton updated with png

holistudio committed:
Navbar updated with Brand and MenuButton components

holistudio committed:
ChatbotMessage Component updated

holistudio committed:
ChatbotMessageDialogBubble Component updated

holistudio committed:
chatbot message CSS

holistudio committed:
hompage with navbar and chatbot message components

holistudio committed:
Main.js update for display tests

holistudio committed:
Added a few CSS webpack plugins

holistudio committed:
Component directly references css file

This isn't exactly modular but seems to be enough to get a simple website up and running.

holistudio committed:
HTML.js references the correct style sheet

holistudio committed:
new individual CSS files for each component created

holistudio committed:
extraneous css files deleted and references removed

holistudio committed:
renamed user reply components to user message components

holistudio committed:
renamed UserMessage.css to UserMessageInput

holistudio committed:
updated UserMessageBox css with class selector

holistudio committed:
Home component and CSS update

holistudio committed:
passing props from Conversation.js to ChatbotMessage component

holistudio committed:
removed conversation page from Home screen

holistudio committed:
navigation styling

holistudio committed:
conversation container width

holistudio committed:
componentDidMount functions combined into one

holistudio committed:
Conversation.js handle change and handle submit functionality with components

IT ALL WORKS NOW OMG

holistudio committed:
update brand.js to prevent nesting <a> elements

holistudio committed:
moved image files to client/public/static

holistudio committed:
installed file and url loader

because of course you need more loaders #webpacklyfe

holistudio committed:
add url loader to webpack config

holistudio committed:
add images as background images of div elements via CSS

yeah, pretty roundabout way of doing things, but it works! until i can figure out a better way, this is it...

holistudio committed:
initial merge with development

holistudio committed:
packages! MOAR PACKAGES!

holistudio committed:
jest css with config

holistudio committed:
Revert "packages! MOAR PACKAGES!"

This reverts commit 49e2eaa4c6cae8469aae39ceee4a71846cea0d6c.

holistudio committed:
style changes

but now the code doesn't work D:

project-omnibus commented:
yeyy

project-omnibus reviewed (changes requested):
Hey i tried to deploy this branch on heroku and it didn't work because it wouldn't find mini-css-extract-plugin during the webpack build. It's not in the package.json here. Can you update that?

holistudio committed:
added mini-css back into packages

holistudio committed:
"ignore css" code added back to start.js

this does trigger style warnings but let's ignore this for now.

holistudio committed:
added url-loader back into dev dependencies

holistudio committed:
clicking on icon leads to modal overlay

holistudio committed:
modal overlay styling for conversation component

holistudio committed:
moved modal forms to conversation component

holistudio committed:
message input box styling

holistudio committed:
background image

holistudio committed:
background image from blogger template

holistudio committed:
move conversation component methods and constructor to Home.js (temporary)

holistudio committed:
visual styling of chatbot conversation box

holistudio committed:
chatbot conversation box animation!

holistudio committed:
fixed Conversation component rendered through Home.js

holistudio committed:
conversation styling adjustment

holistudio committed:
conversation overlay animations

holistudio committed:
Conversation handleSubmit error logging

holistudio committed:
removed unnecessary click handlers from conversation.js

holistudio committed:
logging conversations to separate file, conversation.log, for further analysis

holistudio committed:
questionController typo

holistudio committed:
renamed modal variable

holistudio committed:
moved conversation css to home css

holistudio committed:
moved to new computer

holistudio committed:
moved menu css to navigation.css

holistudio committed:
moved menu component to Navigation.js

holistudio committed:
basic menu button and menu added

holistudio committed:
todo's for home.css

holistudio committed:
navigation menu reformat

holistudio committed:
link added to nav menu

holistudio committed:
navmenu recommendation link behavior update

holistudio committed:
empty book recommendations as homepage

holistudio committed:
menu styling

holistudio committed:
Menu's Chat onclick behavior

holistudio committed:
Notification bubble

holistudio committed:
Added chatbot's notification

holistudio committed:
notification styling

holistudio committed:
added buttons to notification message

holistudio committed:
Added buttons to the notification

holistudio committed:
clicking OK on notification activates the chatbot

holistudio committed:
button clicks close the notification

holistudio committed:
key prop assigned to buttons

holistudio committed:
Conversation div remains stationary as you scroll up and down

holistudio committed:
notification bubble remains stationary during scroling

holistudio committed:
books.js returns the entire volumeInfo JSON from google books API

holistudio committed:
notification bubble button text update

holistudio committed:
initial copy of BookSearch.js code to Home.js

Since the homepage is now a showing of recommendations, the Google Books API request code in BookSearch.js is copied over to Home.js

holistudio committed:
empty book display styling

holistudio committed:
store userprofile to local storage

holistudio committed:
add dummy user profile for testing recommendation results

holistudio committed:
attempt to load a Google Books query with dummyProfile keywrods

robert2030ca committed:
home component did mount change

robert2030ca committed:
empty dummyprofile
